docs/CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,55 @@
+Contributor Code of Conduct
```

Comment:
There's a Sphinx plugin that can render markdown.  I will try using that.

Reply:
Made some changes:

* Dropped the commit that made `docs/CODE_OF_CONDUCT.md` a `.rst` document.  Hence the force push.
* Enabled  `commonmark` extension in Sphinx configuration. 
* Renamed `docs/README.md` to `.txt`.  Otherwise Sphinx would warn about it being not included in the toc tree.
* Added `commonmark` to the deps list in tox "docs" environment.